invenio_oauthclient/contrib/cern.py

```diff
@@ -206,9 +220,12 @@ def account_groups(account, resource, refresh_timedelta=None):
         return account.extra_data.get('groups', [])
 
     groups = fetch_groups(resource['Group'])
+    extra_data = fetch_extra_data(resource)
```

Comment:
Also @lnielsen @pamfilos @tiborsimko do you find any case that this should be configurable in order in the future to parse any field in the cern oauth response?

Reply:
@zzacharo I think since we are adding this feature, it's better to have it configurable